src/wp-admin/includes/post.php

```diff
@@ -1815,7 +1815,8 @@ function _admin_notice_post_locked() {
 			$sendback = add_query_arg( 'post_type', $post->post_type, $sendback );
 		}
 
-		$sendback_text = get_post_type_object( $post->post_type )->labels->all_items;
+		$post_type_object = get_post_type_object( $post->post_type );
+		$sendback_text    = $post_type_object ? $post_type_object->labels->all_items : __( 'Go to All Posts' );
```

Comment:
What about the CPT? It also return text `Go to All Posts` if CPT it for work?

Reply:
Hii @mukeshpanchal27 thanks for the review. I have updated my PR. 

Also here we will face a problem of singular and Plural names of CPT. As in the fallback case a post-type object will not be available, and we will be unable to extract the plural name of CPT. 

The idea of adding 's' at the end of the fallback string. 
( This would generate wrong words if someone has a CPT named `Person` as text generated would be `Go to All Persons` which is incorrect ).